com.vscodium.codium-insiders.yaml

```diff
@@ -0,0 +1,144 @@
+id: com.vscodium.codium-insiders
+sdk: org.freedesktop.Sdk
+runtime: org.freedesktop.Sdk
+runtime-version: '23.08'
+command: codium-insiders
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --socket=pulseaudio
```

Comment:
Is pulseaudio actually needed?

Reply:
Yes, some extensions use sounds to notify users